src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -2,6 +2,69 @@
 
 public class SalaryInfo {
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        int salaryFirst = 0;
+        int salarySecond = 0;
+        int salaryThird = 0;
+        String firstName = "";
+        String secondName = "";
+        String thirdName = "";
+
+        String [] splitDataFrom = dateFrom.split("[.]");
+        String [] splitDataTo = dateTo.split("[.]");
+        String [] splitCurrentData = new String[3];
+        String [] splitInputString = new String[4];
+
+        int currentDay = 0;
+        int currentMonth = 0;
+        int toDay = Integer.parseInt(splitDataTo[0]);
+        int fromDay = Integer.parseInt(splitDataFrom[0]);
+        int toMonth = Integer.parseInt(splitDataTo[1]);
+        int fromMonth = Integer.parseInt(splitDataFrom[1]);
+        firstName = names[0];
+
+        for (int i = 1; i < names.length; i++) {
+            if (!firstName.equals(names[i])) {
+                secondName = names[i];
+            }
+            break;
+        }
+        for (int i = 1; i < names.length; i++) {
+            if (!firstName.equals(names[i]) && !secondName.equals(names[i])) {
+                thirdName = names[i];
+                break;
+            }
+        }
+
+        //sort by data
```

Comment:
Don't begin class or method implementation with an empty line. Please remove the unnecessary empty line before the loop.